src/DotNetLightning.Core/Channel/CommitmentsModule.fs

```diff
@@ -439,6 +439,58 @@ module ForceCloseFundsRecovery =
         return obscuredCommitmentNumber
     }
 
+    let tryGetFundsFromRemoteCommitmentTx (localParams: LocalParams)
```

Comment:
nit: It may be good to add an explanation that adding outputs to the returned `TransactionBuilder` is a caller's responsibility.
This also applies to the local counterpart (`tryGetFundsFromLocalCommitmentTx`)

Reply:
I added doc comments to the two functions.